docs/tutorials/grover_with_sampler.ipynb

```diff
@@ -7,17 +7,17 @@
    "source": [
```

Comment:
<p>This explanation is clearer now I think</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>